### ISSUE TITLE
[Feature Request] Containerless proxies

### ISSUE THREAD

**Issue by deAtog** [nginx-proxy/nginx-proxy#1533]
Please consider adding support for generating nginx configuration sections from an optional user configuration file. While it is possible to add proxies for sites by adding additional configuration files, this can lead to a discrepancy in security protocols and proxy settings among manually added and dynamically generated proxies. Additionally, the Lets Encrypt companion is unable to locate the appropriate configuration files/sections for these hosts in order to correctly create/mange the certificates.

**Comment by tkw1536** [comment]
I would disagree, for now the workaround is probably to use [custom proxy-wide-configuration](https://github.com/nginx-proxy/nginx-proxy#proxy-wide), with a `server` block inside.
